docs/fast_data/concepts/data_loading.mdx

```diff
@@ -0,0 +1,472 @@
+---
+id: data_loading
+title: Fast Data Loading
+sidebar_label: Data Loading
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+__Data Loading__ is a fundamental process to introduce information from the Systems of Record:
+through Change Data Capture (CDC) solutions, data streams will flow into your Event Streaming Platform of choice
+and then be processed by the Fast Data architecture of the runtime.
+
+In this section, we will explore the two main operations that impacts Data Loading.
+
+## Initial Load
+
+In Fast Data, __Initial Load__ operations are made to synchronize the Systems of Record's tables with the Fast Data Projections.
+
+### Overview
+
+Beside the standard real-time data stream, the Initial Load operation is carried out by the CDC, 
+which will produce ingestion messages for each record of the tables that have been chosen to be sent to Fast Data. 
+
+:::caution
+Usually CDCs will stop the capture of real time events to the source systems during an Initial Load:
+this means that other tables not involved in the operation will not receive updates during the operation.
+:::
+
+### Scenarios
+
+This operation is needed when:
+
+- the Fast Data is initialized, to make it consistent with the source system;
+- there has been some _disalignment_ between the source system and Fast Data, due to either
+  infrastractural problems or misconfigurations
+
+### Best Practices
+
+Since Initial Load may involve tables having huge size, is necessary to prepare your infrastructure accordingly 
+and stop the generation of Single Views.
+
+:::tip
+If you need to Bootstrap either Fast Data services or infrastructural component, you can check out [this section](/fast_data/faq/bootstrapping_fast_data.md#what-should-be-the-required-resources-to-perform-an-initial-load). 
+:::
+
+Before an Initial Load takes place, you have to prepare your [Fast Data architecture](/fast_data/concepts/architecture.md#putting-it-all-together) accordingly.
+
+<Tabs groupId='fast-data-architecture'>
+
+<TabItem value="event-driven" label="Event-Driven Architecture" default>
+
+First, you have to check that ingestion topics have been correctly defined for the Projections that needs to be populated.
+
+Then, if there exists a Single View Trigger Generator that consumes the messages produced into the `pr-update` topics of
+the projections, you have to check that message consuming is not up-and-running (either by using Runtime Management or setting replicas to zero).
+
+In this way, during the ingestion phase, strategies will not be executed. Once the ingestion procedure has been completed, you can restart
+the SVTG, so that strategies will generate the single view identifiers accordingly.
+
+##### Using Runtime Management
+
+With Runtime Management, is possible to manage Initial Load operations without altering your configuration.
+
+From the Control Plane frontend, you have to check first that the Domain Pipeline that will consume messages
```

Comment:
Alternatively, here we may put a link to runtime management fast data section with control plane info

Reply:
link added [here](https://github.com/mia-platform/documentation/pull/1391/commits/730e00d01bbf09241b3c687c05224456843836a7)